templates/about.html

```diff
@@ -3,21 +3,76 @@
 {% block title %}About{% endblock title %}
 
 {% block main %}
-	<h2>About</h2>
-	<p>Here you will find an overview of all credentials currently issued by or known to the Privacy by Design Foundation. This overview is generated from the XML files of the <a href="glossary.html#scheme-manager">schemes</a> listed below. For more information, see the <a href="glossary.html">glossary</a>.</p>
-	<h3>Schemes</h3>
-	<dl class="schememanagers">
-		{% for schememgr in index %}
-		<dt><a href="{{ schememgr.id }}.html">{{ schememgr.name[LANG] }}</a></dt>
-		<dd>
-			<p>{{ schememgr.description[LANG] }}</p>
-			<p>The following issuers are contained in this scheme:</p>
-			<ul>
-				{% for issuerId, issuer in schememgr.issuers.items() | sort %}
-				<li><a href="{{ issuer.identifier }}.html">{{ issuer.name[LANG] }}</a></li>
-				{% endfor %}
-			</ul>
-		</dd>
-		{% endfor %}
-	</dl>
+	<h2>Yivi Credentials System</h2>
+	<p>Yivi is a privacy-friendly identity solution, that can be used to issue and verify identity materials or credentials. 
+	Yivi credentials are unique, in that they consist of attributes that can be independently disclosed, 
```

Comment:
Yivi credentials are unique in that they consist of attributes that can be independently disclosed, allowing the rest of the identity to **remain** **undisclosed**. An attribute describes a single aspect of an identity. The cards below illustrate how the various components of the **Yivi trust system** are addressed throughout the Yivi ecosystem. A more in-depth explanation can be found in the <a href="https://docs.yivi.app/">Yivi Documentation</a>.

Rather credential scheme or trust system I suppose.